src/core/MOM.F90

```diff
@@ -1131,6 +1135,17 @@ subroutine step_MOM_dynamics(forces, p_surf_begin, p_surf_end, dt, dt_thermo, &
     call disable_averaging(CS%diag)
   endif
 
+  !OBC segment data update for some fields can be less frequent than others
```

Comment:
The intentation and spacing in this block of code do not comply with MOM6 standards, as articulated at https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide.